packages/atomic/src/components/common/search-box/wrapper.spec.ts

```diff
@@ -0,0 +1,34 @@
+import {html, render} from 'lit';
```

Comment:
I'd also test the part and @focusout, and also probably try passing children to ensure they're getting rendered.